docs/spark/tutorials/databricks-deployment.md

```diff
@@ -9,161 +9,211 @@ ms.custom: mvc
 
 # Deploy a .NET for Apache Spark application to Databricks
 
-This tutorial teaches how to deploy a .NET for Apache Spark application to Databricks.
+This tutorial teaches you how to deploy your app to the cloud through Azure Databricks, an Apache Spark-based analytics platform with one-click setup, streamlined workflows, and interactive workspace that enables collaboration.
 
 In this tutorial, you learn how to:
 
 > [!div class="checklist"]
->
-> * Prepare Microsoft.Spark.Worker
-> * Publish your Spark .NET app
-> * Deploy your app to Databricks
-> * Run your app
+> Create an Azure Databricks workspace.
+> Publish your .NET for Apache Spark app.
+> Create a Spark job and Spark cluster.
+> Run your app on the Spark cluster.
 
-## Prerequisites
+Before you start, do the following tasks:
```

Comment:
Why remove the prerequisites section? it's also part of the tutorial

Reply:
I think it was removed unintentionally.